doc/howtos/booting-and-installation.rst

```diff
@@ -351,14 +356,23 @@ the partition tables are wiped out.
 Running Ostro OS in a VirtualBox\* VM
 ======================================
 
-You can run an Ostro OS image within a VirtualBox virtual machine by using the pre-built ``.vdi`` file found
+You can run an Ostro OS image within a VirtualBox virtual machine by using the pre-built ``.ova`` file found
```

Comment:
we have for latest automated builds. e.g.  https://download.ostroproject.org/builds/ostro-os/latest/images/intel-corei7-64/ostro-image-noswupd-intel-corei7-64.dsk.vdi.zip
but maintaining those names in different places might be not good thing to do.